setup.py

```diff
@@ -1,15 +1,36 @@
 #!/usr/bin/env python
 from setuptools import setup, find_packages, Extension
 import glob
+import os
+
+
+class MissingPkgconfig:
+    """Raise an exception only when trying to convert it to a string.
+
+    This allows commands like setup.py clean to work even when pkgconfig
```

Comment:
Backticks around `setup.py clean` for readability ?

Reply:
Done